### PR TITLE
Update macOS instructions with cmake

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Following up on a suggestion+PR from @kaskavel.

##### Component Name

area/docs

### CODE REVIEW

File: packaging/installer/methods/macos.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ xcode-select --install
 ```
 
 Click **Install** on the Software Update popup window that appears. Then, use the same terminal session to use Homebrew
-to install some of Netdata's prerequisites.
+to install some of Netdata's prerequisites. `cmake` is necessary only if you need to claim the node to Netdata Cloud.
```
Comment (underhood) [inline packaging/installer/methods/macos.md:53]: I would maybe reword it as `cmake is necessary in case you want to use Netdata Cloud functionality` or `you can skip/omit cmake in case you do not want to use Netdata Cloud` but that is subjective. LGTM

**Review by underhood — DISMISSED** [review]

**Review by kaskavel — DISMISSED** [review]

File: packaging/installer/methods/macos.md (hunk 1 above)
Comment (OdysLam) [reply packaging/installer/methods/macos.md:53]: I prefer not to assume that people know what claiming or netdata cloud are.
```suggestion
to install some of Netdata's prerequisites. `cmake` is necessary only if you need to [claim](https://learn.netdata.cloud/docs/cloud/get-started) the node to [Netdata Cloud](https://netdata.cloud/cloud).
```

**Review by OdysLam — CHANGES_REQUESTED** [review]

**Comment by underhood** [comment]
@OdysLam && guys what I worry here is people will look at that and read Cloud and immediately disable out of reflex never actually learning until they Claim nothing is ever sent.

**Review by kaskavel — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]

**Comment by krital** [comment]
Just to clarify we need both lib tool (added yesterday) and cmake

**Comment by joelhans** [comment]
> Just to clarify we need both lib tool (added yesterday) and cmake

Of course. This PR pre-dated the lib update from yesterday, so I just had to fix the merge conflict to make sure they both end up in the docs.

**Review by kaskavel — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]
